python/cuml/manifold/t_sne.pyx

```diff
@@ -248,13 +263,12 @@ class TSNE(Base):
             warnings.warn("n_iter = {} might cause TSNE to output wrong "
                           "results. Set it higher.".format(n_iter))
         if metric.lower() != 'euclidean':
-            warnings.warn("TSNE does not support {} but only Euclidean. "
-                          "Will do in the near future.".format(metric))
+            warnings.warn("TSNE does not support {} (only Euclidean).".format(
```

Comment:
I believe there's an open issue for UMAP (and possibly TSNE) to support the other metrics available in `NearestNeighbors`. Canyou reference that issue here in a TODO?